CMakeLists.txt

```diff
@@ -185,6 +185,18 @@ target_link_libraries("IntegrationStrattoMosaic"  Cubature)
 target_link_libraries("IntegrationStrattoMosaic"  ${cuba_dir}/libcuba.a)
 install(TARGETS "IntegrationStrattoMosaic" RUNTIME DESTINATION bin)
 
+add_executable("IntegrationStrattoMosaicSG" "simulations/IntegrationStrattoMosaicSG.cpp")
+target_link_libraries("IntegrationStrattoMosaicSG" ${LIBS} ${CUBALIBS} ${STRATTOLIBS})
+target_link_libraries("IntegrationStrattoMosaicSG"  Cubature)
+target_link_libraries("IntegrationStrattoMosaicSG"  ${cuba_dir}/libcuba.a)
+install(TARGETS "IntegrationStrattoMosaicSG" RUNTIME DESTINATION bin)
+
+add_executable("IntegrationStrattoMosaicSGZernike" "simulations/IntegrationStrattoMosaicSGZernike.cpp")
+target_link_libraries("IntegrationStrattoMosaicSGZernike" ${ZERNIKELIBS} ${LIBS} ${CUBALIBS} ${STRATTOLIBS})
+target_link_libraries("IntegrationStrattoMosaicSGZernike"  Cubature)
+target_link_libraries("IntegrationStrattoMosaicSGZernike"  ${cuba_dir}/libcuba.a)
```

Comment:
I would change ${cuba_dir}/libcuba.a to Cuba, see the version in branch develop.